src/components/appContext/index.js

```diff
@@ -1,5 +1,5 @@
-import React, { Component } from 'react'
-import PropTypes from 'prop-types'
+// @flow
+import * as React from 'react'
```

Comment:
curious as to why you've chosen to use * as React, instead of just importing React. 

Also, i've noticed that you've removed { Component } from the imports too. If this is specifically a style preference, then it's probably best to revert since the rest of the site seems to be destructuring Component from the react package. 

If I'm missing something let me know, but we should definitely keep consistency

Reply:
This is a somewhat annoying change required by flow to work optimally

https://flow.org/en/docs/react/components/#toc-class-components

In short
> We import React as a namespace here with `import * as React from 'react'` instead of as a default with `import React from 'react'`. When importing React as an ES module you may use either style, but importing as a namespace gives you access to React’s utility types.